fluent/src/bundle.js

```diff
@@ -76,13 +66,19 @@ export default class FluentBundle {
   }
 
   /*
```

Comment:
Make that `/**` so that esdoc picks this up? Same for `hasMessage`.

Reply:
Good catch!